release_notes/ocp-4-11-release-notes.adoc

```diff
@@ -2652,6 +2652,218 @@ You can view the container images in this release by running the following comma
+=====  Recovering clusters after a failed update
+
+For {sno}, you can use the {cgu-operator-first} Operator to create a backup of a current deployment before an {product-title} version update. If the update fails, you can use the backup to return the cluster to its pre-update state.
+For more information, see xref:../scalability_and_performance/cnf-talm-for-cluster-upgrades.html#talo-backup-feature-concept_cnf-topology-aware-lifecycle-manager[Creating a backup of cluster resources before upgrade].
+
+[id="ocp-4-11-RAN-disabling-chronyd"]
+=====  Disable chronyd in the PolicyGenTemplate CR
+
+You must disable `chronyd` if you update to {product-title} 4.11 from earlier versions. To disable `chronyd`, add the following line in the `[service]` section under `.spec.profile.data` of the `TunedPerformancePatch.yaml` file. The `TunedPerformancePatch.yaml` file is referenced in the group `PolicyGenTemplate` CR:
+
+[source,yaml]
+----
+[service]
+service.chronyd=stop,disable
+----
+
+For more information, see xref:../scalability_and_performance/ztp-vdu-validating-cluster-tuning.adoc#recommended-cluster-kernel-configuration[Recommended cluster kernel configuration].
+
+[id="ocp-4-11-ran-known-issues"]
+==== {product-title} 4.11 RAN known issues
+
+* After deploying a {sno} cluster with the RAN DU profile, an error `openvswitch: cpu_id mismatch with handler threads` is continuously generated in the Open vSwitch kernel logs. (link:https://issues.redhat.com/browse/OCPBUGSM-46165[*OCPBUGSM-46165*])
+
+* If secure boot is currently disabled and you try to enable it using ZTP, the cluster installation does not start. When secure boot is enabled through ZTP, the boot options are configured before the virtual CD is attached. Therefore, the first boot from the existing hard disk has the secure boot turned on. The cluster installation gets stuck because the system never boots from the CD. (link:https://issues.redhat.com/browse/OCPBUGSM-45085[*OCPBUGSM-45085*])
+
+* If an invalid subscription channel is specified in the subscription policy that is used to perform a cluster upgrade, the {cgu-operator-full} indicates a successful upgrade right after the policy is enforced because the `Subscription` state remains `AtLatestKnown`. (link:https://issues.redhat.com/browse/OCPBUGSM-43618[*OCPBUGSM-43618*])
+
+* If the SRIOV-FEC Operator is installed through separate policies and the `CatalogSource` reuses the name of a default catalog source, the Operator installation might fail due to conflicts with the management of the default catalog sources. To avoid this issue the SRIOV-FEC `CatalogSource` CR should be added to the `common-config-policy` and the Operator subscription should be added to the `common-subscriptions-policy`. As a workaround when using separate policies to install the SRIOV-FEC Operator, you must ensure that the `CatalogSource` for this Operator is uniquely named. (link:https://issues.redhat.com/browse/OCPBUGSM-39859[*OCPBUGSM-39859*])
```

Comment:
```suggestion
* If the SRIOV-FEC Operator is installed through separate policies and the `CatalogSource` reuses the name of a default catalog source, the Operator installation might fail due to conflicts with the management of the default catalog sources. To avoid this issue, the SRIOV-FEC `CatalogSource` CR should be added to the `common-config-policy` and the Operator subscription should be added to the `common-subscriptions-policy`. As a workaround when using separate policies to install the SRIOV-FEC Operator, you must ensure that the `CatalogSource` for this Operator is uniquely named. (link:https://issues.redhat.com/browse/OCPBUGSM-39859[*OCPBUGSM-39859*])
```